base/applications/taskmgr/lang/zh-HK.rc

```diff
@@ -438,7 +438,7 @@ BEGIN
     IDS_MSG_WARNINGCHANGEPRIORITY "警告：更改此處理程序的優先順序可能導致系統不穩定。\n你確定要更改優先順序嗎？"
     IDS_MSG_TRAYICONCPUUSAGE "CPU 使用情況： %d%%"
     IDS_STATUS_MEMUSAGE "記憶體使用情況： %s ／ %s (%d%%)"
-    IDS_STATUS_CPUUSAGE "CPU 使用情況： %3d%%"
+    IDS_STATUS_CPUUSAGE "CPU: %3d%%"
```

Comment:
```suggestion
    IDS_STATUS_CPUUSAGE "CPU： %3d%%"
```

Reply:
zh-ZK: Hermes suggested the chinese colon here (although it is not structurally used everywhere else in this file).